### PR TITLE
Add option to show selected option in SearchMultiSelectInput

### PR DESCRIPTION
**Pull request by samshara**
### This PR ensures that there are
- [x] No unnecessary console logs
- [x] No unwanted comments
- [x] No typos
- [x] `yarn typecheck` shows no issue
- [x] `yarn lint` shows no issue

### This PR adds one or more new components / utils

- [x] Add the newly created component and its props to exports
- [x] Write story for newly created component(s)
- [x] No empty files

### This PR fixes or updates existing components / utils

- [ ] Update / add story to reflect the changes
- [ ] Doesn't introduce breaking changes

### Beta release vXX.XX.XX-beta.YY
- [ ] Add a commit for beta release

### CODE REVIEW

File: src/components/SearchMultiSelectInput/index.tsx [hunk 1]
```diff
@@ -38,6 +40,8 @@ export type Props<
     sortFunction?: (options: O[], search: string, labelSelector: (option: O) => string) => O[];
     onSearchValueChange?: (value: string) => void;
     onShowDropdownChange?: (value: boolean) => void;
+    selectedOptionContainerClassName?: string;
+    showSelectedList?: boolean;
```
Comment (tnagorra) [inline src/components/SearchMultiSelectInput/index.tsx]: Let's change the prop name to something like `listShown`
@AdityaKhatri @frozenhelium any thoughts?

File: src/components/SearchMultiSelectInput/index.tsx [hunk 2]
```diff
@@ -253,31 +261,66 @@ function SearchMultiSelectInput<
...
-            onDropdownShownChange={handleChangeDropdown}
-            focused={focused}
-            onFocusedChange={setFocused}
-            focusedKey={focusedKey}
-            onFocusedKeyChange={setFocusedKey}
-            persistentOptionPopup
-            nonClearable={false}
-            hasValue={isDefined(value) && value.length > 0}
-        />
+        <>
```
Comment (tnagorra) [inline src/components/SearchMultiSelectInput/index.tsx:265]: We should either wrap with div or SelectInputContainer should support some prop for this

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/components/SearchMultiSelectInput/index.tsx (hunk 2 above)
Comment (samshara) [reply src/components/SearchMultiSelectInput/index.tsx:265]: @AdityaKhatri, @tnagorra what do you suggest we do? Is it a good idea if we pass it as a child to SelectInputContainer?

File: src/components/SearchMultiSelectInput/index.tsx (hunk 1 above)
Comment (AdityaKhatri) [reply src/components/SearchMultiSelectInput/index.tsx]: `selectionListShown`

File: src/components/SearchMultiSelectInput/index.tsx (hunk 2 above)
Comment (AdityaKhatri) [reply src/components/SearchMultiSelectInput/index.tsx:265]: @frozenhelium what do you think?

Comment (frozenhelium) [reply src/components/SearchMultiSelectInput/index.tsx:265]: @AdityaKhatri  I have no idea what we're trying to achieve here

Comment (tnagorra) [reply src/components/SearchMultiSelectInput/index.tsx:265]: Should the SelectInputContainer or InputContainer support showing a component just below it?

We wouldn't need a wrapper if it is supported.

**Review by tnagorra — APPROVED** [review]

File: src/components/SearchMultiSelectInput/index.tsx (hunk 2 above)
Comment (subinasr) [reply src/components/SearchMultiSelectInput/index.tsx:265]: @AdityaKhatri how do we handle this ? We can use this component in `lead copy to other projects`.

**Pull request merged**